index.js

```diff
@@ -193,17 +193,24 @@ Shopify.prototype.request = function request(uri, method, key, data, headers) {
  * Updates GraphQL API call limits.
  *
  * @param {String} throttle The status returned in the GraphQL response
+ * @param {Number} requestedQueryCost The requested cost from the GraphQL query
+ * @param {Number} actualQueryCost The actual cost from the GraphQL query
  * @private
  */
-Shopify.prototype.updateGraphqlLimits = function updateGraphqlLimits(throttle) {
+Shopify.prototype.updateGraphqlLimits = function updateGraphqlLimits(
+  throttle,
+  requestedQueryCost,
+  actualQueryCost
```

Comment:
I would pass only `extensions.cost` here, it includes everything. Also please add `actualQueryCost` and `requestedQueryCost` in the object on line 58.